app/views/deals/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with(model: [@group, @deal]) do |form| %>
+  <% if deal.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(deal.errors.count, "error") %> prohibited this deal from being saved:</h2>
+
+      <ul>
+        <% deal.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.text_field :name, class:'form-field', placeholder:'Name' %>
+
+  <%= form.text_field :amount, class:'form-field', placeholder:'Amount' %>
+
```

Comment:
- [ ] I would recommend adding the feature to select the `categories` here in the form as per requirements. Please update this.

![Screenshot from 2022-04-30 00-17-18](https://user-images.githubusercontent.com/22434660/166007619-724ba4ad-f7a5-4407-8a23-c33dfc19da61.png)